must-call-checker/src/main/java/org/checkerframework/checker/mustcall/NotOwning.astub

```diff
@@ -41,13 +41,13 @@ class SocketChannel extends AbstractSelectableChannel implements ByteChannel, Sc
 
 class ServerSocketChannel extends AbstractSelectableChannel implements NetworkChannel {
     protected ServerSocketChannel(SelectorProvider arg0);
-    static ServerSocketChannel open() throws IOException;
+    @MustCallChoice static ServerSocketChannel open(@MustCallChoice ServerSocketChannel this) throws IOException;
     int validOps();
-    ServerSocketChannel bind(SocketAddress arg0) throws IOException;
-    ServerSocketChannel bind(SocketAddress arg0, int arg1) throws IOException;
+    @MustCallChoice ServerSocketChannel bind(@MustCallChoice ServerSocketChannel this, SocketAddress arg0) throws IOException;
+    @MustCallChoice ServerSocketChannel bind(@MustCallChoice ServerSocketChannel this, SocketAddress arg0, int arg1) throws IOException;
     <T> ServerSocketChannel setOption(SocketOption<T> arg0, T arg1) throws IOException;
     @MustCallChoice ServerSocket socket(@MustCallChoice ServerSocketChannel this);
-    SocketChannel accept() throws IOException;
+    @MustCallChoice SocketChannel accept(@MustCallChoice ServerSocketChannel this) throws IOException;
```

Comment:
This annotation is wrong - `accept` produces a new socket. See e.g. https://docs.oracle.com/javase/7/docs/api/java/nio/channels/ServerSocketChannel.html#accept().

Reply:
You're right, `accept()` will create a new socket from the client-side.